mchf-eclipse/drivers/ui/lcd/ui_lcd_hy28.h

```diff
@@ -24,6 +24,13 @@
 	#define MAX_Y  240
 #endif
 
+#ifdef TimeDebug
+#define MARKER LCD_D0
+#define MARKER_PIO LCD_D0_PIO
+
+#define Marker_ON  MARKER_PIO->BSRR=MARKER;
```

Comment:
Hi Slawek,
please do not use direct assignments, these are not portable (your code works on H7 and F4 not on F7, ask STM why).
GPIO_SetBits() / GPIO_ResetBits() are our portable macros.

Reply:
OK, i'll remove this code after we end with lcd  optimization.

Reply:
Danilo, what is wrong now? I see in conflicts that it refuses all changes i made.